src/ReactiveUI.Tests/ObservableAsPropertyHelperTest.cs

```diff
@@ -229,6 +229,26 @@ public void ToPropertyShouldFireBothChangingAndChanged()
             Assert.Equal("Baz", resultChanged[1].Value);
         }
 
+        private class LazyPropertyFixture : ReactiveObject
+        {
+            public LazyPropertyFixture()
+            {
+                value = Observable
+                    .Return(new Lazy<int>(() => 20 ))
+                    .ToProperty(this, p=>p.Value, new Lazy<int>(()=>25));
```

Comment:
Given the `Observable.Return` will pipe straight through in this case, should the `initialValue` be used at all? Would it be worth adding a test case around `initialValue` being used first or do the other test cases cover it off (in which case is it worth dropping it here to reduce noise).